doc/modules/mod_muc.md

```diff
@@ -17,9 +17,9 @@ Also `mod_muc_log` is a logging submodule.
 * `history_size` (non-negative integer, default: 20): Room message history to be kept in RAM.
  After node restart, the history is lost.
 * `room_shaper` (atom, default: `none`): Limits per-room data throughput with traffic shaper.
-* `max_room_id` (atom or positive integer, default: `infinite`): Maximum room username length (in JID).
-* `max_room_name` (atom or positive integer, default: `infinite`): Maximum room name length.
-* `max_room_desc` (atom or positive integer, default: `infinite`): Maximum room description length.
+* `max_room_id` (atom or positive integer, default: `infinity`): Maximum room username length (in JID).
+* `max_room_name` (atom or positive integer, default: `infinity`): Maximum room name length.
+* `max_room_desc` (atom or positive integer, default: `infinity`): Maximum room description length.
```

Comment:
The problem is that such docs will be replaced with TOML and the new doc will just say "use the `cfg` one form the previous version". But there it was called `infinite`. This makes it necessary to add the migration info for all such changes and this was the reason I tried to minimize them even though it is tempting to just rush with them.

I don't think this change is bad (I also considered it), but it needs to be documented.

Reply:
I've checked with code. `infinite` and `infinity` are both fine from mod_muc point of view. It just needs to be an atom to pass as an infinite case.
So, both would work for an old config format (unless we are changing code).

It could be a good time to uniform that now.

Reply:
We could add to the migration guide information that `infinite` should not be used though.

Reply:
Ok it's fine then.